### DISCUSSION TITLE
Recrystallization literature and open source code

### DISCUSSION THREAD

**Discussion by KangChenRui** [idaholab/moose#24409] [Q&A Modules: Phase field]
Hi，Everyone！
       I would like to learn about the literature and open source code related to recrystallization. 
       Is there any content on these aspects on MOOSE?
Thank you for your help

Chenrui

**Comment by dschwen** [comment]
We have done recrystallization simulations, where we keep track of a stored energy content in each grain and use the DiscreteNucleation system to nucleate new grains. @amjokisaari has worked on this.

**Comment by permcody** [comment]
https://link.springer.com/article/10.1007/s11837-019-03830-z
